double-press-switch.js

```diff
@@ -0,0 +1,42 @@
+// this script how a classic switch (not a push switch) can be used to to perform different action simple and double 
```

Comment:
Similar comments apply as for the double-press-double-switch.js

Reply:
Also done